gradle-plugins/model-compiler/src/main/java/io/spine/gradle/compiler/Extension.java

```diff
@@ -41,6 +46,8 @@
 // as this is a Gradle extension.
 public class Extension {
 
+    @VisibleForTesting
+    static final String SPINE_BUILD_ARTIFACT_STORAGE_DIR = ".spine";
```

Comment:
I thought we had a single place. Now I don't understand. Let's discuss.

Also, this is a code style issue.